src/applications/simple-forms/20-10206/sass/10206-pa.scss

```diff
@@ -5,3 +5,17 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-modal";
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
 @import "../../../../platform/forms/sass/m-form-confirmation";
+
+// titleUI H3s
+.form-panel legend > h3 {
+  margin-bottom: 0 !important;
+}
+
+// record-selections page
+.form-panel.record-selections {
+  va-checkbox-group[name="root_recordSelections"]
+    legend.usa-legend
+    > span.usa-label__required {
+    display: none !important;
```

Comment:
I'm not sure I understand why the "required" text needs to be remove this way? In what scenario would you not just remove the `required` prop?